x/virtualgroup/types/params.go

```diff
@@ -75,6 +88,13 @@ func (p Params) Validate() error {
 	if err := validateMaxStoreSizePerFamily(p.MaxStoreSizePerFamily); err != nil {
 		return err
 	}
+	if err := validateSwapInValidityPeriod(p.SwapInValidityPeriod); err != nil {
+		return err
+	}
+	if err := validateSwapInValidityPeriod(p.SwapInValidityPeriod); err != nil {
```

Comment:
This is not fixed yet, ==> `validateSPConcurrentExitNum`

Reply:
fixed